pkg/recovery/recovery.go

```diff
@@ -0,0 +1,6 @@
+package recovery
+
+// Recoverer interface for recovering with different implementations
```

Comment:
I think the previous comment or something similar was more helpful here, otherwise it is unclear what we are recovering from or why:
```go
// Recoverer offers the ability to recover from security violation or boot failure
```

Reply:
Done